.github/workflows/drivers.yml

```diff
@@ -367,35 +367,64 @@ jobs:
       with:
         junit-name: 'be-tests-mysql-latest-ee'
 
-  be-tests-oracle-ee:
+  be-tests-oracle-18-4-ee:
     if: github.event.pull_request.draft == false
     runs-on: ubuntu-20.04
     timeout-minutes: 60
     env:
       CI: 'true'
       DRIVERS: oracle
-      MB_ORACLE_TEST_USER: metabase
-      MB_ORACLE_TEST_PASSWORD: ${{ secrets.MB_ORACLE_TEST_PASSWORD }}
-      MB_ORACLE_TEST_SID: ORCL
-      MB_ORACLE_TEST_HOST: ${{ secrets.MB_ORACLE_TEST_HOST }}
+      MB_ORACLE_TEST_HOST: localhost
+      MB_ORACLE_TEST_USER: system
+      MB_ORACLE_TEST_PASSWORD: password
+      MB_ORACLE_TEST_SERVICE_NAME: XEPDB1
+    services:
+      oracle:
+        image: gvenzl/oracle-xe:18.4.0-slim
+        env:
+          ORACLE_PASSWORD: password
+        ports:
+          - "1521:1521"
+    steps:
+    - uses: actions/checkout@v3
+    - name: Test Oracle driver (18.4)
+      uses: ./.github/actions/test-driver
+      with:
+        junit-name: 'be-tests-oracle-18-4-ee'
+
+  be-tests-oracle-21-3-ee:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    env:
+      CI: 'true'
+      DRIVERS: oracle
+      MB_ORACLE_TEST_HOST: localhost
+      MB_ORACLE_TEST_PORT: 2484
+      MB_ORACLE_TEST_SERVICE_NAME: XEPDB1
+      MB_ORACLE_TEST_SSL: true
+      MB_ORACLE_TEST_SSL_USE_TRUSTSTORE: true
+      MB_ORACLE_TEST_SSL_TRUSTSTORE_PATH: './test_resources/ssl/oracle/truststore.p12'
+      MB_ORACLE_TEST_SSL_TRUSTSTORE_OPTIONS: local
+      MB_ORACLE_TEST_SSL_TRUSTSTORE_PASSWORD_VALUE: 'PassworD_#1234'
+      MB_ORACLE_TEST_SSL_USE_KEYSTORE: true
+      MB_ORACLE_TEST_SSL_KEYSTORE_PATH: './test_resources/ssl/oracle/keystore.p12'
+      MB_ORACLE_TEST_SSL_KEYSTORE_OPTIONS: local
+      MB_ORACLE_TEST_SSL_KEYSTORE_PASSWORD_VALUE: 'PassworD_#1234'
       MB_ORACLE_SSL_TEST_SSL: true
-      MB_ORACLE_SSL_TEST_PORT: 2484
-      MB_ORACLE_SSL_TEST_SSL_USE_TRUSTSTORE: true
-      MB_ORACLE_SSL_TEST_SSL_TRUSTSTORE_PATH: './resources/certificates/rds_root_ca_truststore.jks'
```

Comment:
I wonder if this file should be deleted.

Reply:
If it's only for tests, it should be removed. I don't see any references to that path anywhere else in the code, so I would assume it's safe.

Reply:
You see, it's not in `test_resources`, but as you noted, there is no other reference to it in the code base.

Reply:
According to https://github.com/metabase/metabase/tree/master/resources/certificates

> This is a keystore that contains only the RDS root CA. If you try to use it as the JVM-level truststore, you will probably get a lot of SSL related errors, but it should work in isolation for particular connections (ex: Oracle SSL connectivity).

Looks like it was added in https://github.com/metabase/metabase/pull/15078

> Adding RDS root CA truststore file to our resources so that CircleCI (and anyone else, really) can refer to it

so I guess maybe the original intention was that other people could use it as well. However, we never documented it anywhere, so it's probably safe to delete it